app/models/concerns/metadata.rb

```diff
@@ -17,6 +17,10 @@ module Metadata
       index.as :symbol, :stored_searchable # symbol is needed for exact match search in the CollectionFactory
     end
 
+    property :isbn, predicate: RDF::Vocab::Identifiers.isbn do |index|
```

Comment:
Can you put this just on the ETD class? It shoudn't exist for Images or Collections.